corehq/apps/data_interfaces/tests/test_auto_case_updates.py

```diff
@@ -10,7 +10,8 @@
 
 from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
 from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
-from corehq.form_processor.tests.utils import run_with_all_backends, FormProcessorTestUtils, set_case_property_directly
+from corehq.form_processor.tests.utils import (run_with_all_backends, FormProcessorTestUtils,
+    set_case_property_directly, set_parent_case as set_actual_parent_case)
```

Comment:
@gcapalbo: I'm getting an import error on this line when I run the tests with nose. It appears that `set_parent_case` does not exist in [corehq/form_processor/tests/utils.py](https://github.com/dimagi/commcare-hq/blob/20a9b68f49d1ac79f9f2098e8528221b596b2d5a/corehq/form_processor/tests/utils.py), and that leads me to believe that this file is not being found by the django test runner, and therefore the tests in it are not being run.

Here are the logs for the [most recent test run](https://travis-ci.org/dimagi/commcare-hq/builds/127491332) after this change on #11462. I can't find `AutomaticCaseUpdateTest` or `test_match_days_since`, which is a test in this file, in any of them. For my sanity, can you confirm that none of the tests in this file are being run by the django test runner?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/127491334/log.txt
https://s3.amazonaws.com/archive.travis-ci.org/jobs/127491335/log.txt
https://s3.amazonaws.com/archive.travis-ci.org/jobs/127491336/log.txt